### ISSUE TITLE
Analyze DMARC reports and update SPF and DKIM as necessary

### ISSUE THREAD

**Issue by feanil** [openedx/tcril-engineering#17]

**Comment by sarina** [comment]
@feanil does this still need to be addressed?

**Comment by feanil** [comment]
Yes,  I've just created a PR to update some of our email rules that should make this way less urgent and allow us to close it as not-planned for now.  We may eventually want to set this up if we're getting reports of spam coming from our domains but the changes in https://github.com/aximcollaborative/terraform-internal/pull/134 should prevent most issues.  I'll assign this to myself right now and put it in progress.  Once Ed reviews the changes I can close this.

**Comment by feanil** [comment]
The relevant security updates have been applied.  I'll close this in a day or two once we've confirmed that this has not caused any issues.

**Comment by feanil** [comment]
We haven't seen any issues with the applied changes so I think we can mark this as done.  The SPF records are now enforcing and all the DKIM records are in-place.  DMARC has also been updated for all our domains to be collected in one place.  If we want to do further analysis with a 3rd party tool, it shouldn't be too hard to do.
